ioctl/cmd/account/account.go

```diff
@@ -178,6 +178,29 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
 	return response.AccountMeta, nil
 }
 
+// IsSignerExist checks whether signer account is existed
+func IsSignerExist(signer string) bool {
+	addr, err := address.FromString(signer)
+	if err != nil {
+		return false
+	}
+	// find the account in keystore
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+		keystore.StandardScryptN, keystore.StandardScryptP)
+	for _, ksAccount := range ks.Accounts() {
+		if address.Equal(addr, ksAccount.Address) {
+			return true
+		}
+	}
+	// find the account in pem files
+	pemFilePath := sm2KeyPath(addr)
+	_, err = os.Stat(pemFilePath)
+	if err == nil {
+		return true
```

Comment:
return (err == nil)?

Reply:
also, i recall seeing this code (check KeyStore then PEM file) in your SM2 check-in

why it is missed

Reply:
I used to put all account related functions in account.go. I don't why someone put this function in action.go, which is odd and I missed it last time. Now, I have put this function in account.go.